presto-main/src/main/java/com/facebook/presto/sql/analyzer/QueryExplainer.java

```diff
@@ -144,10 +144,10 @@ public String getGraphvizPlan(Session session, Statement statement, Type planTyp
         switch (planType) {
             case LOGICAL:
                 Plan plan = getLogicalPlan(session, statement, parameters, warningCollector);
-                return graphvizLogicalPlan(plan.getRoot(), plan.getTypes(), session);
+                return graphvizLogicalPlan(plan.getRoot(), plan.getTypes(), session, metadata.getFunctionManager());
```

Comment:
Add a TODO: function manager will be removed after `CallExpression` is introduced in `AggregationNode`